examples/baichuan/build.py

```diff
@@ -261,7 +261,7 @@ def parse_arguments():
         default=False,
         action="store_true",
         help=
-        'By default, we use dtype for KV cache. fp8_kv_cache chooses int8 quantization for KV'
+        'By default, we use dtype for KV cache. fp8_kv_cache chooses fp8 quantization for KV'
     )
```

Comment:
by default it 

> args.quant_mode.set_int8_kv_cache()

with int8 device type (host type ?)

with this option on, it sets fp8 dtype for kv cache

> args.quant_mode = args.quant_mode.set_fp8_kv_cache()

The codes also shows that enable fp8 is orthogonal to enable fp8 for kv cache:

```
    if args.int8_kv_cache:
        args.quant_mode = args.quant_mode.set_int8_kv_cache()
    elif args.fp8_kv_cache:
        args.quant_mode = args.quant_mode.set_fp8_kv_cache()
    if args.enable_fp8:
        args.quant_mode = args.quant_mode.set_fp8_qdq()
```

I suggested that :

> By default, we use int8 device type for KV cache. with "fp8_kv_cache" on, the builder chooses fp8 device type quantization for KV cache.